reference-implementation/lib/writable-stream.js

```diff
@@ -300,11 +300,7 @@ function WritableStreamFinishInFlightClose(stream) {
 }
 
 function WritableStreamFinishInFlightCloseInErroredState(stream) {
```

Comment:
I would be OK with inlining WritableStreamFinishInFlightWriteInErroredState and WritableStreamFinishInFlightCloseInErroredState now that they are two-liners and each called twice. Up to you though, I guess they are reasonably semantic operations that can stand on their own.